src/components/Player/Pages/Overview/Overview.jsx

```diff
@@ -238,57 +270,46 @@ Overview.propTypes = {
   location: PropTypes.string,
 };
 
+const getData = (props, gamemode) => {
+  let params = props.location.search;
+  const hasGameMode =
+    props.location &&
+    props.location.search &&
+    props.location.search.indexOf('game_mode') !== -1;
+  if (gamemode === 'turbo' && !hasGameMode) {
+    params = `${props.location.search}&significant=0&game_mode=23`;
+  }
 
-const getData = (props) => {
   if (props.location.search) {
-    props.getPlayerMatches(props.playerId, props.location.search);
+    props.getPlayerMatches(props.playerId, params);
```

Comment:
similarly, i'd delegate the switch to the actions file instead of changing all this